rattler/static/analysis/index.js

```diff
@@ -0,0 +1,199 @@
+$( document ).ready(function() {
+
+        // Die Variablen müssen wegen Jinja in der html-Datei vorbereitet werden!
+
+        // Funktion, um Spalte in 2. Dimension als Zeile auszugeben
+        // https://stackoverflow.com/a/34979219
+        const arrayColumnAsRow = (arr, n) => arr.map(x => x[n]);
```

Comment:
Bitte keine Arrow functions, da diese nicht in jedem Browser implementiert worden sind.